learn/getting_started/algolia_migration.md

```diff
@@ -160,15 +160,15 @@ const BATCH_SIZE = 1000;
 
 Meilisearch's default settings are designed to deliver a fast and relevant search experience that works for most use-cases.
 
-To customize your index settings, we recommend following [this guide](/learn/configuration/settings.md). To learn more about the differences between settings in Algolia and Meilisearch, read on.
+To customize your index settings, we recommend following [this guide](/reference/api/settings.md). To learn more about the differences between settings in Algolia and Meilisearch, read on.
```

Comment:
API reference ≠ a guide, replace with link to settings section of index core concepts page.

In general, when linking from the getting started, we should prefer "explain"-y guides like those in the Learn section. There are of course exceptions, like the below example.